tests/jest/engine/simple-events.test.js

```diff
@@ -7,7 +7,7 @@ const namespaceName = "simpleeventstest"
 
 describe('Test events states behaviour', () => {
     beforeAll(common.helpers.deleteAllNamespaces)
-    afterAll(common.helpers.deleteAllNamespaces)
+    //afterAll(common.helpers.deleteAllNamespaces)
```

Comment:
```suggestion
    afterAll(common.helpers.deleteAllNamespaces)
```

Reply:
I think this is unnessary change.

Reply:
a sorry this was not supposed to be committed.